### PR TITLE
[Native/guide] Add info about checking app permission(s)

### PR DESCRIPTION
**Pull request by flukson**
### Change Description ###

Add information about checking the permission(s) of the application
with the specified app_id.

### API Changes ###

ACR: ACR-1296
http://suprem.sec.samsung.net/jira/browse/ACR-1296

### CODE REVIEW

File: docs/application/native/guides/security/privacy-related-permissions.md [hunk 1]
```diff
@@ -227,6 +227,32 @@ To check whether an application has permission to use a privilege, and to reques
         }
```
[the remark this answers is not in the document]
Comment (flukson) [reply docs/application/native/guides/security/privacy-related-permissions.md]: These remarks are not related to my changes so I cannot fix them in this pull request.

File: docs/application/native/guides/security/privacy-related-permissions.md [hunk 2]
```diff
@@ -227,6 +227,32 @@ To check whether an application has permission to use a privilege, and to reques
...
+
+2. Check whether the application has permission to use the privileges, using the `ppm_check_app_permission()` or `ppm_check_app_permissions()`, in a way similar to `ppm_check_permission()` and `ppm_check_permissions()`. The only difference is that you need to add the app_id as the first argument of the used function. For example:
+
+    ```
+    void check_app_permission() {
+        ppm_check_result_e result;
+        const char *app_id = "org.tizen.camera-app";
+        const char *privilege = "http://tizen.org/privilege/camera";
+
+        int ret = ppm_check_permission(app_id, privilege, &result);
```
[the remark this answers is not in the document]
Comment (flukson) [reply docs/application/native/guides/security/privacy-related-permissions.md]: You are right. I have fixed it.

File: docs/application/native/guides/security/privacy-related-permissions.md [hunk 3]
```diff
@@ -227,6 +227,32 @@ To check whether an application has permission to use a privilege, and to reques
...
+> Checking permissions of the application with the specified app_id is supported from Tizen 5.0. Requesting permissions of the application with the specified app_id is not supported.
+
+To check whether the application with the specified app_id has permission to use the privileges:
+
+1. Declare the `http://tizen.org/privilege/permission.check` privilege in the manifest of the application. If the privilege is missing, the API functions that are designed to check the privileges of the application with the specified app_id, return the `PRIVACY_PRIVILEGE_MANAGER_ERROR_PERMISSION_DENIED` error.
+
+2. Check whether the application has permission to use the privileges, using the `ppm_check_app_permission()` or `ppm_check_app_permissions()`, in a way similar to `ppm_check_permission()` and `ppm_check_permissions()`. The only difference is that you need to add the app_id as the first argument of the used function. For example:
+
+    ```
+    void check_app_permission() {
```
[the remark this answers is not in the document]
Comment (flukson) [reply docs/application/native/guides/security/privacy-related-permissions.md]: Ok.

**Comment by mijin85cho** [comment]
Thanks for the update @flukson.

@rishu-samsung, would you please check the latest version?

File: docs/application/native/guides/security/privacy-related-permissions.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rishu-samsung) [reply docs/application/native/guides/security/privacy-related-permissions.md]: @flukson Thanks for letting me know. No issues, we can implement these points in further modifications.

File: docs/application/native/guides/security/privacy-related-permissions.md [hunk 4]
```diff
@@ -227,6 +227,34 @@ To check whether an application has permission to use a privilege, and to reques
...
+
+        int ret = ppm_check_app_permission(app_id, privilege, &result);
+    }
+    ```
+
+    The result of the call is stored in the variable passed as the second parameter (for available results, see the `ppm_check_result_e` enumeration in [mobile](../../api/mobile/latest/group__CAPI__PRIVACY__PRIVILEGE__MANAGER__MODULE.html#ga41f409d8b9d4c27d41b907bdb0975f0c) and [wearable](../../api/wearable/latest/group__CAPI__PRIVACY__PRIVILEGE__MANAGER__MODULE.html#ga41f409d8b9d4c27d41b907bdb0975f0c) applications).
+
 ## Related Information
 - Dependencies:
   - Tizen 4.0 and Higher for Mobile
```
[the remark this answers is not in the document]
Comment (flukson) [reply docs/application/native/guides/security/privacy-related-permissions.md]: > Please keep the 'h' in small case for the word 'Higher' in both the points. Kindly rephrase as:
> 
> * Tizen 4.0 and higher for Mobile
> * Tizen 4.0 and higher for Wearable

These remarks are not related to my changes so I cannot fix them in this pull request.

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/guides/security/privacy-related-permissions.md]: Hello, @rishu-samsung.
About this, would you please get guide from other Reviewer Members in SRI-B?

This sentences are in common format, used in whole guide pages.
I think you'd possible to see other pages are also using this format for the platform version info at the end.
To get the consistency with those, we should keep current sentence.

**Comment by mijin85cho** [comment]
So many thanks to your review & update for this PR.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**